packages/sdk-router/src/swap/engines/liquidSwapEngine.ts

```
@@ -0,0 +1,260 @@
+    }
+  }
+
+  public async generateRoute(
+    input: RouteInput,
+    quote: LiquidSwapQuote
+  ): Promise<SwapEngineRoute> {
+    if (quote.engineID !== this.id || !quote.data) {
+      logger.error({ quote }, 'LiquidSwapEngine: unexpected quote')
+      return getEmptyRoute(this.id)
+    }
+    let tokens: string[] = []
```

ESLint

[error] 'tokens' is never reassigned. Use 'const' instead.

(prefer-const)